src/languages/css-language.js

```diff
@@ -51,6 +51,34 @@ const blockCloserTokenTypes = new Map([
 	[tokenTypes.RightSquareBracket, "["],
 ]);
 
+/**
+ * Recursively replaces all function values in an object with boolean true.
+ * Used to make objects serializable for JSON output.
+ *
+ * @param {Record<string,any>} object The object to process.
```

Comment:
To match the return type:

```suggestion
 * @param {Record<string,any>|unknown[]|unknown} object The object to process.
```

Alternatively, `unknown` alone should be sufficient.